src/test/java/com/wikia/webdriver/testcases/featuredVideo/FeaturedVideoDesktopTests.java

```diff
@@ -0,0 +1,130 @@
+package com.wikia.webdriver.testcases.featuredVideo;
+
+import com.wikia.webdriver.common.core.Assertion;
+import com.wikia.webdriver.common.core.annotations.Execute;
+import com.wikia.webdriver.common.core.helpers.User;
+import com.wikia.webdriver.common.templates.NewTestTemplate;
+import com.wikia.webdriver.pageobjectsfactory.componentobject.featuredvideo.FeaturedVideoComponentObject;
+
+import org.testng.annotations.Test;
+
+@Test(groups = {"FeaturedVideoDesktop"})
+@Execute(onWikia = "featuredvideo", asUser = User.ANONYMOUS)
+public class FeaturedVideoDesktopTests extends NewTestTemplate {
+
+  @Test
+  public void videoIsPresentOnArticle() {
+    FeaturedVideoComponentObject video = new FeaturedVideoComponentObject()
+        .setAutoplayCookie(false)
+        .openWikiArticle("FeaturedVideo");
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "FeaturedVideo" 9 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)